src/app/pages/abandoned-animal/create-abandoned-animal/create-abandoned-animal.page.html

```diff
@@ -0,0 +1,117 @@
+<app-general-container-component>
+  <h1>Registrar Animal Abandonado</h1>
+</app-general-container-component>
+
+<form [formGroup]="form" (ngSubmit)="submit()">
+  <div class="row">
+
+    <!-- Especies -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Especie</mat-label>
+        <mat-select formControlName="species">
+          @for (s of speciesHttp.speciesList(); track s.id) {
+            <mat-option [value]="s.name">{{ s.name }}</mat-option>
+          }
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <!-- Sex -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Sexo</mat-label>
+        <mat-select formControlName="sex">
+          <mat-option [value]="null">Ninguno</mat-option>
+          @for (s of sexHttp.sexList(); track s.id) {
+            <mat-option [value]="s.name">{{ s.name }}</mat-option>
+          }
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <!-- Age -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Edad estimada</mat-label>
+        <mat-select formControlName="estimatedAge">
+          <mat-option value="Cachorro (menor a 6 meses)">Cachorro (menor a 6 meses)</mat-option>
+          <mat-option value="Joven (6 meses – 2 años)">Joven (6 meses – 2 años)</mat-option>
+          <mat-option value="Adulto (2 a 7 años)">Adulto (2 a 7 años)</mat-option>
+          <mat-option value="Senior (más de 7 años)">Senior (más de 7 años)</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <!-- Physical -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Condición física</mat-label>
+        <mat-select formControlName="physicalCondition">
+          <mat-option value="Saludable">Saludable</mat-option>
```

Comment:
Esto de valores quemados ah si, no esta bien.

Minimo deberia por lo menos ponerlos en un enumerador y usar la estructura de control For para iterar la lista 

deberia de quedar algo como 
```suggestion
       @for (physicalCondition of physicalConditionList)
          <mat-option [value]="physicalCondition">{{ physicalCondition  }}</mat-option>
```

La lista claramente deberia de haber venido de la tabla. (servicio)

Oh bien en el enum en el front end y luego hacer algun tipo de 

```typescript
readonly physicalConditionList = signal<AlgunEnumerador[]>([...Object.values(AlgunEnumerador)])
```

ese signal tendria los valores del enumerador como una lista y luego los iteraria en la estructura de control del html.

Idealmente eso seria un servicio.